Samples/DynamicDependencies/Basic/cpp-console-unpackaged/DynamicDependenciesBasic/DynamicDependenciesBasic.cpp

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright(c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// DynamicDependenciesBasic.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+#include <windows.h>
+#include <MddBootstrap.h>
+
+// Used to demo usage of a Windows App SDK api
+#include "winrt\Microsoft.Windows.System.Power.h"
```

Comment:
prefer the `<>` style of `#include`, see cpp core guidelines for why

Reply:
for some reason I get compile errors trying to switch to <>. This header is generated from cppwinrt and the winappsdk winmds.